synapse/lib/task.py

```diff
@@ -115,6 +115,17 @@ def pack(self):
 
         return pask
 
+    def packv2(self):
+        return {
+            'iden': self.iden,
+            'name': self.name,
+            'info': copy.deepcopy(self.info),
+            'tick': self.tick,
+            'user': self.user.iden,
```

Comment:
We'll still have to conditionally plumb this in, since the `Task` structure doesn't strictly require a user to be present.